docs/users-guide/events-and-timelines.md

```diff
@@ -0,0 +1,62 @@
+# Events and timelines
+
+A lot of discussions around data have a moment when someone asks a question related to a specific point in time: "Wait, what's the spike in March again?", or "When did the new widget launch?"
+
+Events and timelines are a way to capture that chronological knowledge and make it available when you need it, in context (that is, when you're viewing a chart). Events are a great way to store institutional knowledge about what happened and when, so people (including yourself three months from now) won't have to figure out (again) why the line chart spiked back in March.
+
+## Events
+
+Events are basically dates + a title + a description + an icon. You can add events to Metabase to show important milestones, launches, or anything else, right alongside your data.
```

Comment:
Can events span a date range? "Dates" is plural here but title, description, and icon are singular, so it's a little unclear from reading